app/models/determination.rb

```diff
@@ -25,6 +26,14 @@ def calculate_total
     self.total = self.fees + self.expenses
   end
 
+  def calculate_vat
```

Comment:
hey @aliuk2012, discussed this with @EdwardAndress but just to show you a local manual test where i zeroed the vat_amount for a determination and then viewed it in show view - i.e. how it would be on gamma. This probably needs conditional logic to not confuse users (or ask Product Management if they will accept this)
<img width="1028" alt="screen shot 2015-12-14 at 12 08 47" src="https://cloud.githubusercontent.com/assets/7016425/11780672/a5d97a30-a25b-11e5-96d3-da7fa1aad60b.png">